src/DotVVM.Framework/Resources/Scripts/global-declarations.ts

```diff
@@ -122,6 +122,9 @@ type DotvvmInitEventArgs = {
     readonly viewModel: any
 }
 
+type DotvvmInitCompletedEventArgs = {
```

Comment:
@tomasherceg Is there anything you would like to be passed as an event argument for the `initCompleted` event?